fetch.bs

```diff
@@ -3848,11 +3848,18 @@ steps:
 <ol>
  <li><p>Let <var>request</var> be <var>fetchParams</var>'s <a for="fetch params">request</a>.
 
- <li><p>Let <var>response</var> be null.
+ <li>
+  <p>Let <var>response</var> be the result of invoking <span>consume a preloaded resource</span>
+  for <var>request</var>.
+
+  <p class="note">This is done in main fetch and not in HTTP-fetch to avoid blocking or race
+  conditions between the networking queue and the document queue. The implication is that if a
+  fetch redirects to a URL that was preloaded, it would not be consumed.</p>
+ </li>
 
- <li><p>If <var>request</var>'s <a>local-URLs-only flag</a> is set and <var>request</var>'s
- <a for=request>current URL</a> is not <a lt="is local">local</a>, then set <var>response</var> to a
- <a>network error</a>.
+ <li><p>If <var>response</var> is null, <var>request</var>'s <a>local-URLs-only flag</a> is set and
+ <var>request</var>'s  <a for=request>current URL</a> is not <a lt="is local">local</a>, then set
+ <var>response</var> to a <a>network error</a>.
```

Comment:
This seems to give the idea that a network URL could return something in a local-URLs-only environment, e.g., SVG-as-image. That seems wrong.

Reply:
This part has been deleted